### DISCUSSION TITLE
Multiple IBMi connections at the same time

### DISCUSSION THREAD

**Discussion by Code1109** [codefori/vscode-ibmi#544] [Ideas]
Within your network, it would nice to have multiple connections to different IBMi systems at the same time. Right now only one is possible but let's say you have your Dev system and a Stage system (both diff machines), and you want to browse one or the other or maybe copy source from Stage to Dev... 

RDi is possible so it should with VSC as well.

**Comment by worksofliam** [comment]
@Code1109 There is no way this will work with VS Code and the current implementation.

The next best thing is to have a second VS Code window open to start a new connection.

**Reply by Code1109** [reply]
@worksofliam bummer... but I'll try that and see if I can copy/paste from one to the other one.
